pkg/hubble/container/ring.go

```diff
@@ -153,9 +166,12 @@ func (r *Ring) LastWrite() uint64 {
 	return atomic.LoadUint64(&r.write) - 1
 }
 
-// read reads the *v1.Event from the given read position. Returns false if
-// that position is no longer available to be read, returns true otherwise.
-func (r *Ring) read(read uint64) (*v1.Event, bool) {
+// read reads the *v1.Event from the given read position. Returns an error if
```

Comment:
`s/read reads the/read the/`

Reply:
I think it's fine as as is; `read` is the method name and by convention the comment starts by the function/method name.

Reply:
I'm not changing the first word, only the phrasing.

`func Delete(userid int)` shouldn't be commented as `Delete deletes the user data`, rather as `Delete the user data`.